modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -48,7 +48,7 @@ The resource name must be unique in a namespace. Modify the resource name if you
 +
 [NOTE]
 ====
-The *Serverless Deployment* option is displayed in the *Import from git* form only if the {ServerlessOperatorName} is installed in your cluster. For further details, refer to the {ServerlessProductName} documentation.
+You can set the default resource preference for Import by browsing the User Preferences page and clicking *Applications* -> *Resource type* field. The *Serverless Deployment* option is displayed in the *Import from git* form only if the {ServerlessOperatorName} is installed in your cluster. For further details, refer to the {ServerlessProductName} documentation.
```

Comment:
```suggestion
You can set the default resource preference for Import by browsing the User Preferences page and clicking *Applications* -> *Resource type* field. The *Serverless Deployment* option is displayed in the *Import from Git* form only if the {ServerlessOperatorName} is installed in your cluster. For further details, refer to the {ServerlessProductName} documentation.
```

Reply:
It depends on the form, but in the image below, it is "Import from Git", so maybe the same here?

Reply:
Done.